__tests__/app.test.js

```diff
@@ -88,4 +88,33 @@ describe('GET /api/articles/:article_id', () => {
             expect(body.msg).toBe('not found')
         })
     })
+})
+describe('GET: /api/articles', () => {
+    test('GET 200: should respond with an array of article objects, each with the correct properties and data types for each property', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            const articles = body
+            articles.forEach((article) => {
+                expect(typeof article.article_id).toBe('number')
+                expect(typeof article.author).toBe('string')
+                expect(typeof article.title).toBe('string')
+                expect(typeof article.topic).toBe('string')
+                expect(typeof article.created_at).toBe('string')
+                expect(typeof article.votes).toBe('number')
+                expect(typeof article.article_img_url).toBe('string')
+                expect(typeof article.comment_count).toBe('number')
+                expect(article.body).toBe(undefined)
+            })
+        })
+    })
+    test('GET 200: Articles should be sorted by descending date order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toBeSortedBy("created_at", {descending: true}, {coerce: true})
+        })
```

Comment:
Excellent second test, I really like the TDD approach here to separate these tests into two like this 🏆 
Line `115` could be cut as it's being covered by the first test ✂️